content/build/cache/backends/gha.md

```diff
@@ -99,6 +101,32 @@ For example:
     cache-to: type=gha,mode=max
 ```
 
+## Avoid rate limiting using the GitHub Actions cache API
+
+The GitHub Actions cache API has a [rate limit](https://docs.github.com/en/rest/using-the-rest-api/rate-limits-for-the-rest-api)
+of 60 requests per hour for unauthenticated requests. To avoid rate limiting,
+you can provide a GitHub token which lets you perform 5000 requests per hour.
+
+To supply your GitHub token and use for cache requests with the Docker build
+action, pass the GitHub token as an environment variable. The token must have
+the `repo` scope.
+
+```yaml
+- name: Build and push
+  uses: docker/build-push-action@v5
+  with:
+    context: .
+    push: true
+    tags: "<registry>/<image>:latest"
+    cache-from: type=gha
+    cache-to: type=gha,mode=max
+    env:
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
We don't handle the github token through env var but with `github-token` input: https://github.com/docker/build-push-action/blob/cfed4e995e12d7bee5002f066f5649e5f1cc059c/action.yml#L107-L110

Token is set by default so user does not need to pass it: https://github.com/docker/build-push-action/blob/cfed4e995e12d7bee5002f066f5649e5f1cc059c/action.yml#L109

Also with https://github.com/docker/build-push-action/pull/1133 we set `repository` and `ghtoken` attributes when `gha` cache type is used so I think we could just say that `repository` and `ghtoken` are set automatically but can be enforced to other values if needed like user might want top rely on PAT and not ephemeral `secrets.GITHUB_TOKEN`.